src/services/devHandler/updateUserScoreOnDailyProcessPhase1.js

```diff
@@ -0,0 +1,29 @@
+const {successResponse, errorResponse} = require('../../utils');
+const {
+  updateUserScoreOnDailyProcessPhase1
+} = require('../../processes/scoring/update-user-score-on-daily-process-phase1');
+const {getDb} = require('../../config/mongodb_conn');
+const {DB_COLLECTION_USER_SCORE} = require('../../processes/scoring-constant');
+
+const updateUserScorePhase1 = async (req, res) => {
+  try {
+    const {process_time, user_ids, last_batch} = req.body;
+    const db = await getDb();
+    const userScoreCol = await db.collection(DB_COLLECTION_USER_SCORE);
+    const result = await updateUserScoreOnDailyProcessPhase1(
+      null,
+      userScoreCol,
+      process_time,
+      user_ids,
+      last_batch
+    );
+    return successResponse(res, 'updateUserScoreOnDailyProcessPhase1 sucesfully', result);
+  } catch (error) {
+    console.log(error);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging the error message directly to the console might not be the best approach for error handling. Consider using a logging library that provides more flexibility and control over log management. Also, it would be better to include more context in the log message.

```diff
- console.log(error);
+ logger.error(`Failed to update user score on daily process phase 1: ${error}`);
```
In this case, `logger` could be an instance of a logging library such as Winston or Bunyan.


<!-- This is an auto-generated comment by OSS CodeRabbit -->